lib/shift.rb

```diff
@@ -48,10 +48,7 @@ def key_split(key = nil)
     if key == nil
        @generated_key
     else
-      @generated_key = key.to_i
-      require "pry"; binding.pry
+      @generated_key = key
```

Comment:
when I converted the key to an integer to early, I was having problems accessing that first zero.